cypress/e2e/admin/administrateMigration.feature

```diff
@@ -18,9 +18,22 @@ Feature: Admin Users - To administrate the Migration process.
     When I click on agree migration button
     Then I see the migration is active field
     Then I see the end migration button
+    Then I see the migration mandatory switch is not checked
 
+  Scenario: Admin toggles ldap sync for school in migration
+    Then I see the sync during migration switch is visible and not checked
+    When I toggle the sync during migration switch
+    Then I see the sync during migration switch is checked
 
-  Scenario: Admin ends the migration
+  Scenario: Admin stops the migration
     When I click on end migration button
+    Then I see the end of migration information title
+    Then I see the end of migration information text
+    Then I see the end migration information link href is blog.niedersachsen.cloud/umzug
```

Comment:
I think it would be better if you use examples here because it's a test data. and pass that test data to method

Reply:
so you mean the description without the last part "href is...."?

Reply:
yes, and pass this href value from scenario to method and in method assert against it whether we have the particular href or not. You can have a look at this 

https://cucumber.io/docs/gherkin/reference/#scenario-outline

I'm open to it as I think it would be good if we use this way the values so in end it would look like this 

```
Then I see the end migration information link href is '<href>'

Examples:
| href |
| blog.niedersachsen.cloud/umzug |
```